lib/kennel/template_variables.rb

```diff
@@ -23,7 +23,10 @@ def validate_template_variables(data, key)
       end.compact
       bad = queries.grep_v(/(#{variables.map { |v| Regexp.escape(v) }.join("|")})\b/)
       if bad.any?
-        invalid! "queries #{bad.join(", ")} must use the template variables #{variables.join(", ")}"
+        invalid!(
+          "queries #{bad.join(", ")} must use the template variables #{variables.join(", ")}\n" \
+          "If that is not possible, add `validate_template_variables: -> { false } # query foo in bar does not have baz tag`"
```

Comment:
shoukd this have just been `validate: -> { false }`

Reply:
I did validate_template_variables on purpose to leave room for more future validations